src/LibYear.Lib/PackageVersionChecker.cs

```diff
@@ -30,18 +29,22 @@ public static IEnumerable<Result> AwaitResults(IEnumerable<Task<Result>> results
             return tasks.Select(t => t.Result);
         }
 
-        public async Task<Result> GetResultTask(string packageName, NuGetVersion installed)
+        public async Task<Result> GetResultTask(string packageName, PackageVersion installed)
         {
             var versions = _versionCache.ContainsKey(packageName) ? _versionCache[packageName] : _versionCache[packageName] = await GetVersions(packageName);
             var current = versions.FirstOrDefault(v => v.Version == installed);
-            var latest = versions.FirstOrDefault(v => v.Version == versions.Where(m => !m.Version.IsPrerelease && m.Released >= current?.Released).Max(m => m.Version));
+            var latest = versions.FirstOrDefault(v => v.Version == versions.Where(m => !m.Version.IsPrerelease && m.IsListed).Max(m => m.Version));
```

Comment:
:clap: Good catch ignoring unlisted packages, but I think we actually want to do this filtering in `GetVersions` (as a `.Where()` on line 47)

Reply:
That way our class doesn't even need `IsListed`, because by definition all releases are listed / published (part of the reason behind the rename of `VersionInfo`) -- we may want to rename `GetVersions` => `GetPublishedVersions` to note this specificity.

Reply:
I considered this, but thought of one reason why we wouldn't want to do that - when somebody has a version "installed" that isn't listed anymore.  That could cause problems because the current version would end up `null` when it doesn't really need to be.  Then again, if current is `null`, we probably want to alert the user to the fact that there's a problem instead of just telling them how far out of date they are.

Reply:
Yeah, good point about having an unpublished version currently installed -- let's call `PublishedRelease` just `Release` then, and `IsListed` can be called `Published`, and we can keep the logic as is.

Thanks again for working through all this together!

Reply:
I made that change, except that I called the field `IsPublished` to make it clear it's a boolean,